src/software/ai/ai_wrapper.cpp

```diff
@@ -13,7 +15,7 @@ void AIWrapper::onValueReceived(World world)
 
 void AIWrapper::runAIAndSendPrimitives()
 {
-    if (Util::DynamicParameters->getAIConfig()->RunAI()->value())
+    if (Util::DynamicParameters->getAIControlConfig()->RunAI()->value())
```

Comment:
Should this use config?
```suggestion
    if (config->RunAI()->value())
```

Reply:
This uses the `AIControl` config, which is different from the `AIConfig`. I agree we should use that here, but I think it's out-of-scope for this PR.